# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: third_party/dav1d/src/x86/itx_ssse3.asm
# Commit: abcd7426fb63
# Full Hash: abcd7426fb63f078b3d5e2566c28baa1dd923616
# Author: Alex Chronopoulos <achronop@gmail.com>
# Date: 2019-04-23 09:53:27
# Description:
#   Bug 1546070 - Update dav1d from upstream to f8cac8c. r=TD-Linux
#   
#   The imported version is just one commit over 0.2.2 and solves the crash for this bug.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D28343
# ==============================================================================

diff -r bb714fa13ef9 -r abcd7426fb63 third_party/dav1d/src/x86/itx_ssse3.asm
--- a/third_party/dav1d/src/x86/itx_ssse3.asm	Mon Apr 22 20:41:38 2019 +0000
+++ b/third_party/dav1d/src/x86/itx_ssse3.asm	Mon Apr 22 19:48:05 2019 +0000
@@ -5130,7 +5130,7 @@
     ret
 
 
-cglobal inv_txfm_add_identity_identity_32x32, 4, 6, 8, 16*4, dst, stride, coeff, eob, tx2
+cglobal inv_txfm_add_identity_identity_32x32, 4, 6, 8, 16*5, dst, stride, coeff, eob, tx2
     %undef cmp
 
     mov                     r4, 2
